### PR TITLE
#145: add searchStr to store, move search go slot header

### PR DESCRIPTION
**Pull request by lapinlapin**
#### What's this PR do?
Add search to header slot
#### Where should the reviewer start?
Dashboard.vue
#### How should this be manually tested?
http://localhost:8080/#/
#### What are the relevant tickets?
no tickets
#### Screenshots (if appropriate)
image

#### URL
Provide a link to your deploy from http://staging-ui.trusty.apasia.tech:8080/145-market-search

### CODE REVIEW

File: src/views/Dashboard.vue [hunk 1]
```diff
@@ -21,6 +21,17 @@
           :expanded="true"
           class="lg:w-1/2"
           title="markets">
+          <div
```
Comment (roma219) [inline src/views/Dashboard.vue]: Let's move this whole block to separate `MarketsSearch` component. Or actually it seems that .search-wrapper and .tickers-search containers are not needed at all. So you can rename `SearchInput` to `MarketsSearch` and put all the vuex communications and methods into this separate component too. So in Dashboard it is gonna be just `<MarketsSearch slot="header"/>`

**Review by roma219 — CHANGES_REQUESTED** [review]
All good. Only needs to encapsulate everything about search into separate `MarketsSearch` component (see comment).

File: src/views/Dashboard.vue (hunk 1 above)
Comment (lapinlapin) [reply src/views/Dashboard.vue]: fixed

**Review by roma219 — CHANGES_REQUESTED** [review]
Search Input shoud not be hidden when there is a search string active in it:
image
image

**Review by roma219 — APPROVED** [review]
👍

**Pull request merged**